test/lint/check-pr-description.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
+# Copyright (c) 2020 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+import os
+import re
+import subprocess
+import sys
+import json
+import codecs
+from urllib.request import Request, urlopen
+from urllib.error import HTTPError
+
+# External tools (can be overridden using environment
+GIT = os.getenv('GIT', 'git')
+
+def git_config_get(option, default=None):
+    '''
+    Get named configuration option from git repository.
+    '''
+    try:
+        return subprocess.check_output([GIT, 'config', '--get', option]).rstrip().decode('utf-8')
+    except subprocess.CalledProcessError:
+        return default
+
+def get_response(req_url, ghtoken):
+    req = Request(req_url)
+    if ghtoken is not None:
+        req.add_header('Authorization', 'token ' + ghtoken)
+    return urlopen(req)
+
+def retrieve_json(req_url, ghtoken):
+    '''
+    Retrieve json from github.
+    Return None if an error happens.
+    '''
+    try:
+        reader = codecs.getreader('utf-8')
+        return json.load(reader(get_response(req_url, ghtoken)))
+    except HTTPError as e:
+        error_message = e.read()
+        print('Warning: unable to retrieve pull information from github: %s' % e)
+        print('Detailed error: %s' % error_message)
+        return None
+    except Exception as e:
+        print('Warning: unable to retrieve pull information from github: %s' % e)
+        return None
+
+def retrieve_pr_info(pull, ghtoken):
+    req_url = "https://api.github.com/repos/bitcoin/bitcoin/pulls/"+pull
+    return retrieve_json(req_url, ghtoken)
+
+def main():
+    # Get pull request number from Travis environment
+    if 'TRAVIS_PULL_REQUEST' in os.environ:
+        pull = os.environ['TRAVIS_PULL_REQUEST']
+        ghtoken = git_config_get('user.ghtoken')
+    else:
+        assert False, "No pull request number found, PR description could not be checked"
+
+    # Receive pull information from github
+    info = retrieve_pr_info(pull, ghtoken)
+    if info is None:
+        sys.exit(1)
+
+    body = info['body'].strip()
+
+    # Good enough username regex
+    gh_username = r"/\B@([a-z0-9](?:-?[a-z0-9]){0,38})/gi"
+    assert not bool(re.search(gh_username, body)), "Please remove any GitHub @-prefixed usernames from your PR description"
+
+    html_comment_start = r"/<!--/"
+    assert not bool(re.search(html_comment_start, body)), "Please remove the pull request template from your PR description"
+    html_comment_end = r"/-->/"
```

Comment:
See my comment just two lines above. By the way, in Python the prefix "r" has nothing to do with regexs per se, but just indicates raw strings, i.e. no need to escape special characters, but our search string doesn't contain any of those anyway.

```suggestion
    html_comment_end = "-->"
```

Reply:
Fixed!